Sources/JWTKeychain/JWTKeychainUser/JWTKeychainUser+JWTKeychainUserType.swift

```diff
@@ -45,7 +45,7 @@ extension JWTKeychainUser: JWTKeychainUserType {
         }
     }
 
-    public func convertToPublic() -> Public {
-        return .init(email: email, name: name)
+    public func convertToPublic(on db: DatabaseConnectable) throws -> Future<Public> {
```

Comment:
Is DatabaseConnectable also a Container? Is this the best abstraction level?

Reply:
It was a basic worker. I'm not sure where to get both. Should I just take in the request?

Reply:
Yeah, That's the most versatile one. A bit restrictive maybe but I don't see when we shouldn't use request (outside of tests, but even there it's relatively easy).

Reply:
Sounds good.

Reply:
@siemensikkema done